packages/shared/src/lib/featureManagement.ts

```diff
@@ -8,6 +8,10 @@ export class Features<T extends FeatureValue = string> {
 
   static readonly SubmitArticle = new Features('submit_article');
 
+  static readonly EngagementLoopJuly2023Companion = new Features(
+    'engagement_loop_july2023_companion',
+  );
```

Comment:
Features was still based on flagsmith as our experiment platform, I'm not sure you still need it for Growthbook values

Reply:
As per [DR requirements](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/578617368/WT-1528+Engagement+loops#Experiment-setup), this was specified to be added to the `Features` class -- however, I agree that we can safely move away from it, at the same time, having these constants here does not hurt either?

Reply:
Since Growthbook is something new, rather we find a new way of using them.
So maybe use the Features class, but a new way of defining them.

Think Growthbook should be ok with simply enum type somewhere maybe?

Reply:
Have found a way to still keep the Features. The hook is in the link below.

Sample usage:
```
const searchValue = useFeature(Features.Search);
```

The hook would utilize the default value we set from the original class. 

https://github.com/dailydotdev/apps/pull/2046/files#diff-8e32a1b02c9379ceef35c485660f1fbde721268a14e781f7dde68d706fafa48fR145-R148

Reply:
👍  thanks @sshanzel , I've added the equivalent `useFeatureIsOn` to this PR: https://github.com/dailydotdev/apps/pull/2039/files#diff-8e32a1b02c9379ceef35c485660f1fbde721268a14e781f7dde68d706fafa48fR146-R147